.importignore

```diff
@@ -1,25 +1,2 @@
-node_modules/*
```

Comment:
Why did this file get wiped? Aren't we still importing plugins at the end using ./import.sh?

Reply:
Yes, but we're directly importing the profiler/data directory, which means we don't need to explicitly ignore all the others.